uta_tools/data_sources/transcript_mappings.py

```diff
@@ -0,0 +1,198 @@
+"""The module for Transcript Mappings."""
+import csv
+from typing import Dict, List, Optional
+from uta_tools import TRANSCRIPT_MAPPINGS_PATH, LRG_REFSEQGENE_PATH
+
+
+class TranscriptMappings:
+    """The transcript mappings class."""
```

Comment:
It could be useful to provide examples in the :param: section of the docstring for the get_ methods

Reply:
Ya i need to update all the docs (especially ones from variation-norm), but I will do this in a separate issue.